npm/webpack-dev-server/src/makeWebpackConfig.ts

```diff
@@ -69,6 +69,12 @@ function modifyWebpackConfigForCypress (webpackConfig: Partial<Configuration>) {
     }
   }
 
+  // To prevent styling files to be tree shaken by webpack, we set sideEffects: true ensuring that
+  // they are not unintentionally dropped during testing in production mode.
+  if (webpackConfig?.module?.rules) {
+    webpackConfig?.module?.rules.push({ exclude: /node_modules/, test: /\.(s[ac]ss$|less$|styl$|css$)/i, sideEffects: true })
```

Comment:
Any reason for the `exclude`? Does this have an affect on css files that are imported from `node_modules` in the support file?

Reply:
No particular reason excluding the node-modules. I just wanted to ensure that webpack is not bundling node_modules and hence I added this. This will have an affect on css files imported from `node_modules` and should be removed. An oversight on my end.